bin/mailsync

```diff
@@ -43,6 +43,8 @@ case "$(uname)" in
 		# remember if a display server is running since `ps` doesn't always contain a display
 		pgrepoutput="$(pgrep -ax X\(\|org\|wayland\))"
 		displays="$(echo "$pgrepoutput" | grep -wo "[0-9]*:[0-9]\+" | sort -u)"
+		[ -z $displays ] && [ -d /tmp/.X11-unix ] && displays=$(cd /tmp/.X11-unix && for x in X*; do echo ":${x#X}"; done)
+
 		notify() { [ -n "$pgrepoutput" ] && for x in ${displays:-0:}; do
```

Comment:
the default really should be `0:` or `:0`? in my pc it is `:0`, or system default DISPLAY is `:0.0`
// not sure the different much, and though probably not impact something since `$displays` mostly had value.

Reply:
I'm also confused about it, my default display is also `:0.0` and `0:` doesnt work on me.
@LukeSmithxyz are we sure that the fallback display should be `0:` instead of `:0` ? Don't know if some systems use it.